test/FunctionalTests/Microsoft.OData.Core.Tests/ScenarioTests/UriParser/SelectExpandFunctionalTests.cs

```diff
@@ -1667,6 +1667,96 @@ public void ExpandWithNavigationPropCountWithSearchOptionWorks()
             Assert.NotNull(expandedCountSelectItem.SearchOption);
         }
 
+        // $expand=navProp/fully.qualified.type/$count

```

Comment:
Can we add tests for fully.qualified.type being an alias?

Reply:
@mikepizzo I am not sure what you mean. Kindly clarify.
cc @gathogojr

Reply:
From the spec, parameter aliases are only used for `$filter` or `$orderby` https://docs.oasis-open.org/odata/odata/v4.01/odata-v4.01-part2-url-conventions.html#sec_ParameterAliases

Reply:
Sorry; i meant using a (namespace alias)[http://docs.oasis-open.org/odata/odata-csdl-xml/v4.01/odata-csdl-xml-v4.01.html#sec_Alias] to qualify the type name, rather than the full namespace name.

For example:

<Schema Namespace="fully.qualified" Alias="alias">
...
</Schema>

$expand=navProp/alias.type

Reply:
@mikepizzo I have added the tests